WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
+  NSError *error;
+  BOOL isKbdNotPresent =
+  [[[[FBRunLoopSpinner new]
+     timeout:5]
+    timeoutErrorMessage:@"The keyboard cannot be dismissed. Try to use custom approach instead."]
+   spinUntilTrue:^BOOL{
+     XCUIElement *foundKeyboard = [[FBApplication fb_activeApplication].query descendantsMatchingType:XCUIElementTypeKeyboard].element;
+     return !(foundKeyboard.exists && foundKeyboard.hittable);
```

Comment:
because this approach is supposed to be faster (_fb_isVisible_ requires XCUIElement instance to be resolved, which is an expensive operation) and will not potentially cause "cannot read attributes" issue, which we all like so much :).